### DISCUSSION TITLE
ModuleNotFoundError: No module named 'colorama'

### DISCUSSION THREAD

**Discussion by JL-EU** [Homebrew/discussions#4883] [Everyday usage]
### Output of `brew config`

```shell
OMEBREW_VERSION: 4.1.18
ORIGIN: https://github.com/Homebrew/brew
HEAD: bf8991f0d777fe8e33105f0cd7bf1e5993eed60a
Last commit: 7 days ago
Core tap JSON: 05 Nov 12:28 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
HOMEBREW_SORBET_RUNTIME: set
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_blizzard_avalanche
Clang: 14.0.3 build 1403
Git: 2.39.2 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 8.1.2 => /usr/bin/curl
macOS: 13.6-arm64
CLT: 15.0.0.0.1.1694021235
Xcode: 14.3
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Some installed formulae are deprecated or disabled.
You should find replacements for the following formulae:
  openssl@1.1
```

### Description of issue

Hi,
I have installed ESPHOME using HOMEBREW on an arm based MacAir. This worked fine until I updated ESPHOME and now I get the following error message when compiling: ModuleNotFoundError: No module named 'colorama'.
When trying to install colorama I get error: Warning: No available formula with the name "colorama". Did you mean colormake?
How can I get colorama installed in the homebrew enviroment? Thanks.

**Comment by konnano** [comment]
mkdir ~/.pip
vim ~/.pip/pip.conf # Write the following two lines
​
[global]
break-system-packages = true

cd /opt/homebrew/Cellar/python@3.12/3.12.0/bin
./python3.12 -m pip install --upgrade pip
./python3.12 -m pip install colorama
